src/couch_httpd_auth.erl

```diff
@@ -346,10 +346,10 @@ handle_session_req(#httpd{method='GET', user_ctx=UserCtx}=Req, _AuthModule) ->
                 ]}},
                 {info, {[
                     {authentication_db, ?l2b(config:get("couch_httpd_auth", "authentication_db"))},
-                    {authentication_handlers, [auth_name(H) || H <- couch_httpd:make_fun_spec_strs(
-                            config:get("httpd", "authentication_handlers"))]}
```

Comment:
The way `authentication_handlers` were implemented is inconsistent anyway. The chttpd app has them hardcoded. It also doesn't set the value for `UserCtx#user_ctx.handler`. So `authenticated` field is completely missing if called on clustered interface. I'll think about it more. What would be your suggestions? Any hints?